stats/opentelemetry/client_metrics.go

```diff
@@ -150,7 +150,11 @@ func (h *clientStatsHandler) TagRPC(ctx context.Context, info *stats.RPCTagInfo)
 	var labels *istats.Labels
 	if labels = istats.GetLabels(ctx); labels == nil {
 		labels = &istats.Labels{
-			TelemetryLabels: make(map[string]string),
+			// The defaults for all the per call optional labels that this
+			// OpenTelemetry component currently supports.
```

Comment:
I don't think so. I thought since we discussed how all per call operations get this context, to just unify the mechanism to this. Perhaps I can specify this.

Reply:
"Per call label from a plugin that executes on callpath" discussed offline.